pillarbox-ui/src/main/java/ch/srgssr/pillarbox/ui/widget/player/SphericalSurface.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) SRG SSR. All rights reserved.
+ * License information is available from the LICENSE file.
+ */
+package ch.srgssr.pillarbox.ui.widget.player
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.viewinterop.AndroidView
+import androidx.media3.common.Player
+import androidx.media3.exoplayer.video.spherical.SphericalGLSurfaceView
+import ch.srgssr.pillarbox.ui.extension.playerErrorAsState
+
+/**
+ * Render the [player] content on a [SphericalGLSurfaceView].
+ *
+ * @param player The player to render.
+ * @param modifier The modifier to be applied to the layout.
+ */
+@Composable
+fun SphericalSurface(player: Player, modifier: Modifier = Modifier) {
+    val playerError by player.playerErrorAsState()
+    if (playerError != null) {
```

Comment:
Since we check the error state here, should we provide a way for the integrator to provide a content to display in case of error?
Or do we assume that he will take care of it outside of this method?

Reply:
Good point, should we take care for integrators player error and displaying SurfaceView ? Most of the time integrators would display an error message on top of the player view, so I bet we could remove error handle from low-level composable view.

Reply:
> I bet we could remove error handle from low-level composable view.

I'm fine going that way 🙂